scripts/populate.js

```diff
@@ -3,7 +3,17 @@ var Chatty = (function(originalChatty){
 	var outputString = "";
 	var messagesArray = [];
 
-	
+	Chatty.getMessage = function () {
+		return messagesArray;
+	}
+
+	Chatty.newMessage = function () {
```

Comment:
Remember that getMessages is supposed to accept two arguments, the ID of the element that will contain the created message and a string that will make up the body of the created message. For the sake of an example let's say we used "elId" for element ID (that looks odd without serifs) and "msgString" for the message's body. Line 10 would then look something like    Chatty.getMessage = function (elId, msgString) {